fighthealthinsurance/utils.py

```
@@ -20,6 +20,9 @@
 from requests.exceptions import RequestException
 from markdown_strings import esc_format
 
+from loguru import logger
```

Ruff (0.8.2)

Redefinition of unused `logger` from line 16

Remove definition: `logger`

(F811)